keylime/payloads/keylime-controller/local_action_node_failover.py

```diff
@@ -1,27 +1,23 @@
 import os
 import asyncio
+import subprocess
 
-async def execute(revocation):
-    """Node fail over script
+"""Node fail over script
 
-    This script will be sent to the keylime_agent running on a machine
-    which has an OpenShift client (oc) installed and configured.
+This script will be sent to the keylime_agent running on a machine
```

Comment:
To which keylime_agent will this be sent? The one running on a node that is not compromised, right?

Reply:
Yeah, this one runs only on the keylime-controller, which is not in the oc cluster and thus hopefully more secure. The actions sent to the potentially compromised machine are basically removing the dummy `/tmp/keylime-approved` file which is just used as a marker action of sorts.